plugins/generator-1.18.2/forge-1.18.2/procedures/world_send_chat.java.ftl

```diff
@@ -1,5 +1 @@
-if (!world.isClientSide()) {
-	MinecraftServer _mcserv = ServerLifecycleHooks.getCurrentServer();
-	if (_mcserv != null)
-		_mcserv.getPlayerList().broadcastMessage(new TextComponent(${input$text}), ChatType.SYSTEM, Util.NIL_UUID);
-}
\ No newline at end of file
+if (!world.isClientSide() && world.getServer() != null) world.getServer().getPlayerList().broadcastMessage(new TextComponent(${input$text}), ChatType.SYSTEM, Util.NIL_UUID);
```

Comment:
This should still be a two-liner with indentation (if and then new line for statement)